warp10/src/main/java/io/warp10/script/ext/capabilities/Capabilities.java

```diff
@@ -0,0 +1,68 @@
+//
+//   Copyright 2020  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.ext.capabilities;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import io.warp10.script.WarpScriptStack;
+
+/**
+ * This class is there only to prevent the capabilities map from being accessible
+ * on the stack after a STACKATTRIBUTE call so capabilities cannot be added manually.
+ *
+ */
+public class Capabilities {
```

Comment:
You should also add a `put` method, currently in `CAPADD` to be consistent.

Reply:
Getting capabilities is something meant to be done by functions, adding (put) capabilities is only performed by CAPADD.